src/ui/src/components/data-table/data-table.tsx

```diff
@@ -148,6 +148,9 @@ const useDataTableStyles = makeStyles((theme: Theme) => createStyles({
     maxHeight: '100%',
     width: '100%',
     lineHeight: `${ROW_HEIGHT_PX}px`,
+    '& a': {
+      whiteSpace: 'nowrap', 
+    },
```

Comment:
I think setting the `pre` above to `nowrap` will work well here too - it's not just links that are affected.

Good catch, and thanks for the contribution!

Reply:
Hello, ok, I've pushed changes.